build/get-nrfjprog.js

```diff
@@ -79,6 +79,7 @@ const PLATFORM_CONFIG = {
         // When changing this, remember to also update the nrfjprog version in installer.nsh
         url: 'https://raw.githubusercontent.com/NordicSemiconductor/pc-nrfjprog-js/v1.1.0/nrfjprog/nRF5x-Command-Line-Tools_9_7_1_Installer.exe',
         destinationFile: path.join(DOWNLOAD_DIR, 'nrfjprog-win32.exe'),
+        instructions: "WARNING: You must manually install the latest nRF5x command line tools in this platform. The installer has been left at " + DOWNLOAD_DIR
```

Comment:
Maybe "on" this platform, but rather don't hurt the feelings of Windows users, their life is sad already :)
Could also be just "The installer can be found / is ..." or something, it's not like it was forgot there.

Reply:
The point is convey that this only happens ~in~on windows, not on linux/mac. A dev like me might jump between linux (or mac) an win, and wonder why I have to do the thing only ~in~on one platform.

Let me rephrase a bit.